cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -968,7 +972,7 @@ public void getWorkflowSchedules(HttpRequest request, HttpResponder responder,
         specList.add(entry.getValue());
       }
     }
-    responder.sendJson(HttpResponseStatus.OK, specList);
+    responder.sendJson(HttpResponseStatus.OK, GSON.toJsonTree(specList));
```

Comment:
Nope. netty-http's `HttpResponder` uses its own gson instance (created using `new Gson()`) which doesn't register our codecs, so we need to use our own gson instance.